scd4j/src/main/java/com/datamaio/scd4j/conf/Configuration.java

```diff
@@ -156,21 +160,14 @@ private synchronized String buildCurrentExecutionPath() {
 			SimpleDateFormat df = new SimpleDateFormat("yyyyMMddHHmmss");
 			
 			Path workspace = workspace();
-			Path config = getConfig();
-			String fileName;
-			if(config!=null) {
-				fileName = workspace.relativize(config).toString().replace("config/", "");
-			} else {
-				fileName = workspace.toString().replace("config/", "");
-			}
-			
+			String fileName = this.getConfig().getFileName().toString();
 			if(fileName.endsWith(".conf")) {
 				fileName = fileName.replace(".conf", "");
 			}
 			if(fileName.endsWith(".properties")) {
 				fileName = fileName.replace(".properties", "");
 			}
-			String module = getModule().getFileName().toString();
+			final String module = workspace.relativize(this.getModule().toAbsolutePath()).toString().replace(Configuration.MODULES_FOLDER + File.separator, "");
```

Comment:
I used multi-level module folders and I need a better content visualization (for files and folders) for the folders "log" and "backup" , which reflect that modules configuration. If I use only getModule().getFileName() for that, the contents of "log" and "backup" folders didn't reflect those organization (modules subfolders). For exemple, I used this organization:

modules
- systemA
-- moduleA
-- moduleB
-- moduleC
- systemB
-- moduleD
- systemC
-- moduleE

log
- systemA
-- moduleA
--- file.log
-- moduleB
--- file.log
-- moduleC
--- file.log
- systemB
-- moduleD
--- file.log
- systemC
-- moduleE
--- file.log

backup
- systemA
-- moduleA
--- installed files
-- moduleB
--- installed files
-- moduleC
--- installed files
- systemB
-- moduleD
--- installed files
- systemC
-- moduleE
--- installed files

Without my code, the log and backup folders make sense only if I put the modules as directly folders of "modules" folder.